python/tests/test_stats.py

```diff
@@ -426,14 +542,20 @@ def verify(self, ts, reference_sets, focal=None):
         A2 = tsutil.genealogical_nearest_neighbours(ts, focal, reference_sets)
         A3 = ts.genealogical_nearest_neighbours(focal, reference_sets)
         A4 = ts.genealogical_nearest_neighbours(focal, reference_sets, num_threads=3)
+        A5 = windowed_genealogical_nearest_neighbours(ts, focal, reference_sets)
         assert np.array_equal(A3, A4)
         assert A1.shape == A2.shape
         assert A1.shape == A3.shape
         assert np.allclose(A1, A2)
         assert np.allclose(A1, A3)
+        if not np.any(np.isnan(A5)):
+            assert np.allclose(A1, A5)
+            assert np.allclose(A5, A2)
+            assert np.allclose(A5, A3)
```

Comment:
Might as well check the non-nans.
```suggestion
        mask = ~np.isnan(A5)
        assert np.sum(mask) > 0 or ts.num_edges == 0
        assert np.allclose(A1[mask], A5[mask])
        assert np.allclose(A5[mask], A2[mask])
        assert np.allclose(A5[mask], A3[mask])

```